src/consumer/consumer.js

```diff
@@ -38,7 +38,7 @@ function consumerPromise({id, init}) {
 	return new Promise(() => {
 		const queue = new Queue(init);
 
-		if (id !== 0 && isNotDefined(id)) {
+		if (id !== 0 && !id) {
```

Comment:
Can simplify this further if you can make it so that the worker id is never 0.

Reply:
Yeah, I have this in mind. I'll make a separate relevant PR once everything gets settled down.